src/core/TrainPart.hpp

```diff
@@ -21,6 +20,7 @@ namespace godot {
             bool _commands_registered = false;
 
         protected:
+            // ReSharper disable once CppHidingFunction
```

Comment:
specific linter configuration. how about others?

Reply:
Clang-Tidy have not reported anything as well. This might be old linter configuration (before adding your .clang-tidy). which I have forgot to remove I'll see if it is still relevant there or not.